Content.Client/Lobby/LobbyState.cs

```diff
@@ -69,6 +69,13 @@ protected override void Startup()
                 _lobby.CharacterPreview.UpdateUI();
             };
 
+            //wire up the anonymous class for our Show Round End button:
+            _lobby.ShowRoundSummaryButton.OnPressed += _ =>
+            {
+                ClientGameTicker cgtTickingManager = new ClientGameTicker();
+                cgtTickingManager.DisplayRoundEndSummary(ClientGameTicker._roundEndContainer._message);
+            };
```

Comment:
You should not be constructing entitysystems like this.

Reply:
so previously, I used the same instance of the _window object across different parts of the application, using static, to ensure there was no added memory/processing overheads and unnecessary objects, but some other maintainers/contributors told me to redo it. This is the newer version of the change.

Do you advise I go back to using the same instance of the object - by using static objects, or do you have something else in mind, like moving the initialization of the ClientGameTicker out of the anonymous class, so that a new instance of it isn't created on every click?